torch_xla/csrc/tensor_util.cpp

```diff
@@ -670,17 +661,13 @@ at::Tensor MakeTensorFromXlaLiteral(const xla::Literal& literal,
   }
 }
 
-bool TensorCompare(const at::Tensor& t1, const at::Tensor& t2) {
```

Comment:
@JackCaoG Leave the API "memory less". Just call TensorMemoryCompare() and add a comment about why.
I feel this to be a temp solution, as the compare of two aten tensors should be deterministic, fast-math or not.